### ISSUE TITLE
suggestions / questions for tweaking the design

### ISSUE THREAD

**Issue by KeLaiFu** [JonathanBedrava/eurorack-cv-led-strip-controller#2]
I got my PCBs back, and they work great, however, something I found is that the effect of mixing 3 different gate inputs into one LED strip doesn't really create a distinctive display, obviously, when a red pulse and a blue pulse hit at the same time it then becomes purple etc. After scratching my head a while I joined the switch pins on the jacks to the signal pins, and now it kinda works as a 3 channel colour mixer if you put just one input in, and I think it still works as normal if you add additional inputs on the other jacks, which I think works quite well as an additional option. 

Is this likely to be putting any additional strains on the circuit first of all? I have basically demonstrated the limits of my technical knowledge with this post, so I would welcome any feedback if this is a bad idea, or if there are better ways to achieve the same effect.

**Comment by JonathanBedrava** [comment]
Great question! And I have an answer. The short version: There is no problem but _only if you cut the current traces leading to the switch pins on the PJ301BM audio jacks!_ This can be accomplished easily with a hobby knife of similar. 

Longer answer: The reason this must be done is that the pinout of the audio jacks in the schematic is incorrect. The switch pins are currently connected to ground for some reason. Ooops! This probably also accounts for why the intensity of the strip is fairly low when used as intended since common ground is not established an any unconnected jack mixes in ground, lowering the overall voltage of all other inputs. But do not connect the tips to the switch pin without first cutting the switch pins' traces to ground. Otherwise, by connecting the tips to the switch, you are at risk of creating a short.

A more thorough fix might be to cut the traces to the switch pins and wire the ground pins (on the left and right of the jack) to ground. Test any significant changes out with a power supply and voltage sources you don't have any feelings for.

For bonus points: Once you are _absolutely confident that those ground connections are removed_, you can potentially connect a static voltage source (maybe like 12V divided down to 8V) to the switch and when nothing is plugged in the potentiometer can be used to blend in color.

Thanks again for testing this out. I'll be getting some improvements checked in now that I've given the circuit a good once-over.

**Comment by KeLaiFu** [comment]
I changed the footprint of the jacks to thonkiconns when I gave the kicad file a shuffle, which seems to have inadvertently fixed that ground issue. 
The LEDs certainly didn't feel like they were lacking brightness, judging by the retina burn I had for 10 minutes after testing it for a while.

I am not fully happy with the layout I changed it to, so I was considering another shuffle, I will make those switch / tip connections permanent if I do. I'll throw the files up on my github shortly if you want to take a look.

I'm not sure what your plan was with the unfinished triple module on here, but a module with 3 of these with the mixer functionality would be pretty ideal. Would it also be possible for it to accept lower voltage inputs from say an LFO or CV?

**Comment by JonathanBedrava** [comment]
Changing the value of R4-R6 should change amplification. Replacing them with, say, a 50K trimmer might present an opportunity to set an appropriate level of resistance.

The triple module is a project I hope to get back to before too long. I like the idea of more sophisticated inputs. Like maybe adding inputs for R+B, R+G, B+G, attenuators for everything, and perhaps offsets for each channel. This of course means a lot of jacks and knobs, but it would be nity.

**Comment by JonathanBedrava** [comment]
Also, I totally wouldn't mind seeing what changes you've made!

**Comment by KeLaiFu** [comment]
Yeah, I think a more complex module would be certainly better fitting to this, but also I wonder how much rack space the average euroracker is willing to devote to a lighting module. Given the power consumption for these, I think making it possible to house it in its own enclosure with a devoted PSU would be a good option.

On a similar note, I got tired of trying to get a jack/pot layout that I liked on a single PCB and changed to a PCB sandwich design, which I was then easily able to get down to 2HP, now you mention it, there's plenty of room to switch those resistors for trimmers too!

https://github.com/KeLaiFu/Gate-to-LED-controller

I also connected the jack switches as we discussed, I'd appreciate it if you could double-check I didn't do something dumb :)

**Comment by JonathanBedrava** [comment]
I took a look just now and it doesn't give me any concern insofar as I don't see any reason why it would cause any damage along the circuit or the inputs (but continue to be aware of that very long LED strips can draw a lot of current).

**Comment by KeLaiFu** [comment]
sorry to dig up this old topic, but i found my old PCB for this that I had forgotten all about and started trying to add the trimmers to make it more sensitive to weaker signals and after trying a 50k and 20k trimmer, i threw in a 2k resistor, none of which made any noticeable difference. I used a 074 instead of the 84 in the BOM, would this have anything to do with it?
